packages/fluentui/react-docs/src/parseDefaultValue.ts

```diff
@@ -0,0 +1,41 @@
+import * as _ from 'lodash';
+import * as React from 'react';
+
+import { ComponentPropType } from '@fluentui/docs/src/types';
+import { PropItem } from './docgen';
+
+const parseDefaultValue = (Component: React.ComponentType, propDef: PropItem, types: ComponentPropType[]) => {
+  if (Component.defaultProps && _.has(Component.defaultProps, propDef.name)) {
+    const defaultValue = Component.defaultProps[propDef.name];
+
+    if (_.isFunction(defaultValue)) {
+      return defaultValue.name;
+    }
+
+    if (_.isNumber(defaultValue) || _.isString(defaultValue) || _.isBoolean(defaultValue)) {
+      return defaultValue;
+    }
+
+    if (_.isPlainObject(defaultValue) || _.isArray(defaultValue)) {
+      return defaultValue;
+    }
+
+    if (_.isNull(defaultValue)) {
+      return null;
+    }
+
+    throw new Error(`Can't parse a value in "${Component.name}.defaultProps.${propDef.name}"`);
```

Comment:
This can also produce error that should be noted on readme

Reply:
Error message documented a bit more thoroughly in readme